packages/beasties/src/dom.ts

```
@@ -0,0 +1,359 @@
+
+    removeAttribute: {
+      value(name: string) {
+        if (this.attribs != null) {
+          delete this.attribs[name]
+        }
+      },
+    },
+
+    getAttribute: {
+      value(name: string) {
+        return this.attribs != null && this.attribs[name]
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)